components/alert/test/alert.test.js

```diff
@@ -59,11 +59,63 @@ describe('d2l-alert', () => {
 		it('calling preventDefault on close action should prevent alert from closing', async() => {
 			const alert = await fixture(alertFixture);
 			const closeButton = alert.shadowRoot.querySelector('d2l-button-icon');
-			setTimeout(() => closeButton.click());
 			alert.addEventListener('d2l-alert-close', (e) => {
 				e.preventDefault();
 			});
+			setTimeout(() => closeButton.click());
+			await oneEvent(alert, 'd2l-alert-close');
+			expect(alert.hasAttribute('hidden')).to.be.false;
```

Comment:
I believe it pointed out that `hidden` might not have had time to be set to be `true` with the old structure, so I reworked this.